dap_job_quality/utils/text_cleaning.py

```diff
@@ -76,7 +76,7 @@ def detect_camelcase(text):
     # old patterns: replacement pattern
     # Convert bullet points to fullstops
     "[\u2022\u2023\u25E6\u2043\u2219*]": ".",
-    r"[/:\\]": " ",  # Convert colon and forward and backward slashes to spaces
+    r"[/:\\-]": " ",  # Convert colon, forward and backward slashes and hyphens to spaces
```

Comment:
Why do hyphens need to be converted to spaces?

If we do this, sentences like "Basic salary is between £25k - £40k dependent on experience" and "There will be hybrid work available after the first 1-2 months" become harder to read. That said, we probably only care about this in the context of numeric things like salaries and holiday amount, which I guess we can account for separately (and we can go back to the raw description if necessary for these?)

Reply:
Fair! I was thinking about making it easier to search for things like part time (sometime spelt part-time). But it might be cleaner to leave hyphens in, and see what we can do with fuzzy/ phrase matching (or at the extreme, lists of possible ways to spell part time). I'll change it back